### PR TITLE
Fix highlighting in importlib.metadata docs

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
CC @jaraco / @warsaw

File: Doc/library/importlib.metadata.rst [hunk 1]
```diff
@@ -32,17 +32,17 @@ Overview
...
 
     $ python3 -m venv example
     $ source example/bin/activate
     (example) $ pip install wheel
 
-You can get the version string for ``wheel`` by running the following::
+You can get the version string for ``wheel`` by running the following:
 
-.. highlight:: none
+.. code-block:: pycon
```
Comment (csabella) [inline Doc/library/importlib.metadata.rst:45]: s/pycon/python

Comment (asottile) [reply Doc/library/importlib.metadata.rst:45]: nah, pycon is correct "python console"

there's other examples in the docs:

```console
$ git grep ':: pycon' -- '*.rst'
Doc/extending/embedding.rst:.. code-block:: pycon
Doc/extending/extending.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/faq/extending.rst:.. code-block:: pycon
Doc/faq/extending.rst:.. code-block:: pycon
Doc/faq/windows.rst:.. code-block:: pycon
Doc/faq/windows.rst:.. code-block:: pycon
Doc/howto/logging-cookbook.rst:.. code-block:: pycon
Doc/howto/logging-cookbook.rst:.. code-block:: pycon
Doc/install/index.rst:.. code-block:: pycon
Doc/library/importlib.metadata.rst:.. code-block:: pycon
Doc/library/unittest.mock.rst:.. code-block:: pycon
Doc/library/unittest.mock.rst:.. code-block:: pycon
```

**Comment by jaraco** [comment]
Thank you for doing this.

**Pull request merged**